pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.1</version>
+        <version>1.609</version>
```

Comment:
@jglick I misunderstood your message, I thought you agreed with the change. Anyway I think is a good practice to use LTS baselines since you are relying on a known stable version to develop the plugin.